src/containers/AsideNavigation/useNavigationMenuItems.tsx

```diff
@@ -13,12 +11,15 @@ import {getTenantPath} from '../Tenant/TenantPages';
 
 import i18n from './i18n';
 
+type IconComponent = (props: React.SVGProps<SVGSVGElement>) => JSX.Element;
```

Comment:
Please return previous version. We always use type for `IconData` from gravity.

Reply:
Typing is messed up here, IconData hasn't let me render my icon because of ts errors
I'll try to use IconData somehow but no promises)